Libplanet.Action/State/WorldExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Immutable;
+using Bencodex.Types;
+using Libplanet.Crypto;
+
+namespace Libplanet.Action.State
+{
+    internal static class WorldExtensions
+    {
+        internal static IAccount GetFungibleAssetsAccount(this IWorldState world) =>
+            world.GetAccount(ReservedAddresses.LegacyAccount);
+
+        internal static IAccount GetValidatorSetAccount(this IWorldState world) =>
+            world.GetAccount(ReservedAddresses.LegacyAccount);
+
+        internal static IWorld SetFungibleAssetsAccount(
+            this IWorld world,
+            IAccount account) =>
+            world.SetAccount(ReservedAddresses.LegacyAccount, account);
+    }
+}
```

Comment:
This clashes with `Addresses` by fixing FAV handling scheme.

Reply:
I'll remove this extension, and make reserved account addresses to be set from block policy (from app)

Reply:
Related to https://github.com/planetarium/libplanet/pull/3483